recipes/fa2/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "fa2" %}
+{% set version = "0.3.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fa2-{{ version }}.tar.gz
+  sha256: 9a12e0a7dc228999c605beaae751424ec4a4f60281f44683c0744a72fc9141ef
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+    - scipy
+    - tqdm
+
+test:
+  imports:
+    - fa2
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/bhargavchippada/forceatlas2
+  summary: 'The fastest ForceAtlas2 algorithm for Python (and NetworkX)'
+  license: GPL-2.0-or-later
+  license_file: LICENSE
```

Comment:
Asked upstream to package the license file in the future so we can just use it directly

xref: https://github.com/bhargavchippada/forceatlas2/issues/30